Makefile

```diff
@@ -1975,7 +1975,7 @@ baremetal_prep: ## creates the files to install the operator using olm
 	bash scripts/gen-olm.sh
 
 .PHONY: baremetal
-baremetal: operator_namespace baremetal_prep ## installs the operator, also runs the prep step. Set BAREMETAL_IMG for custom image.
+baremetal: operator_namespace crc_bmo_setup baremetal_prep ## installs the operator, also runs the prep step. Set BAREMETAL_IMG for custom image.
```

Comment:
i guess this means "make baremetal" would only work with crc, which is probably not what we want. open to other ideas.

Reply:
i updated it to work the same way as openstack_prep, so I think it's ok now.